woocommerce/woocommerce
# Replace outdated readme examples of using pnpm

### All Submissions:






### Changes proposed in this Pull Request:

This is mostly just documentation changes of pnpm commands to show that the filter comes first. While the filter coming first is not strictly needed for turbo supported commands (such as `pnpm run build`) I felt it best for consistency to make all the examples the same so people feel more comfortable to remember to put the filter first and always put the command last.

There is also one minor code change to code analyzing where the filter should have been placed first when running a pnpm command from code.

### How to test the changes in this Pull Request:

1. See CI pass for the slight change to analyzer
2. You could just check over the changes here manually and see that all the commands documented show filter before command

### FOR PR REVIEWER ONLY:
